front50-gcs/src/main/java/com/netflix/spinnaker/front50/model/GcsStorageService.java

```diff
@@ -433,7 +441,10 @@ private void writeLastModified(String daoTypeName) {
       try {
           timeExecute(patchTimer, obj_api.patch(bucketName, object.getName(), object));
       } catch (HttpResponseException e) {
-          log.error("writeLastModified failed to update {}\n{}", timestamp_path, e.toString());
+          if (expected) {
```

Comment:
It would always seem like an error to me if this write failed ... not sure it's necessary to gate w/ `expected`?